CHANGELOG.md

```diff
@@ -27,6 +27,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 * The `push.branch-prefix` option was renamed to `git.push-branch-prefix`.
 
+* `jj` will no longer snapshot new files larger than 1MiB by default. This
+  behavior can be customized with the `snapshot.max-new-file-size` config option.
+
```

Comment:
This description can be confusing because it also describes a scenario where jj just silently ignores such files - some people in discord were worried about this

A better wording would include something like "the snapshot operation will fail if there are any new files >1MiB/configured"